includes/adapter/OrderLineFromWooCommerce.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\WooCommerce\Includes\Adapter;
+
+use MyParcelNL\Sdk\src\Model\Fulfilment\OrderLine;
+use WC_Order_Item;
+
+class OrderLineFromWooCommerce extends OrderLine
+{
+    public function __construct(WC_Order_Item $wcOrderItem)
+    {
+        $standardizedDataArray = $this->prepareItemData($wcOrderItem);
+
+        parent::__construct($standardizedDataArray);
+
+    }
+
+    /**
+     * @param \WC_Order_Item $wcOrderItem
+     *
+     * @return array
+     */
+    protected function prepareItemData(WC_Order_Item $wcOrderItem): array
+    {
+        $wcItemData = $wcOrderItem->get_data();
+
+        $data                    = [];
+        $price                   = (int) ($wcItemData['subtotal'] * 100.0);
+        $vat                     = (int) ($wcItemData['subtotal_tax'] * 100.0);
+        $data['price']           = $price;
```

Comment:
Waarom zet je $data niet gewoon in 1x in elkaar zonder een bak extra variabelen?